documentation/installation.md

```diff
@@ -222,9 +222,10 @@ data:
   resourceFilters: "[Event,*,*][*,kube-system,*][*,kube-public,*][*,kube-node-lease,*][Node,*,*][APIService,*,*][TokenReview,*,*][SubjectAccessReview,*,*][*,kyverno,*]"
 ```
 
-By default we have specified Nodes, Events, APIService & SubjectAccessReview as the kinds to be skipped in the default configmap
-[install.yaml](https://github.com/nirmata/kyverno/raw/master/definitions/install.yaml).
+By default we have specified Nodes, Events, APIService & SubjectAccessReview as the kinds to be skipped in the default configuration [install.yaml].
```

Comment:
The configuration is picked from configmap `kyverno:init-config`, it can be modified in the install.yaml or in the actual configmap.
We would want to clarify this.

Reply:
Sure, we'll clarify this.